conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
+Alias /lsmbcss /var/lib/master/css/
+
+<VirtualHost *:80>
+
+  ServerName [SERVERNAME]:80
+  DocumentRoot WORKING_DIR
+
+  <Proxy *>
+    Order Allow,Deny
+    Allow from All
+  </Proxy>
+
+  LogLevel Warn
+
+  <FilesMatch ...>
+    SetHandler  "proxy:pl://localhost:5001"
```

Comment:
@ylavoie can't find any reference to proxy:pl on the web. Do you have a reference for me?